test/e2e/seeder/network-states/eip7702-state/withDelegatorContracts.json

```diff
@@ -0,0 +1,680 @@
506161086000830184613e42565b8181036020830152616119816160d0565b905092915050565b600061613461612f84613a4a565b613a2f565b9050828152602081018484840111156161505761614f6139b9565b5b61615b848285614618565b509392505050565b600082601f830112616178576161776139b4565b5b8151616188848260208601616121565b91505092915050565b600080604083850312156161a8576161a76139aa565b5b600083015167ffffffffffffffff8111156161c6576161c56139af565b5b6161d285828601616163565b92505060206161e385828601615f5c565b9150509250929050565b7f4141333320726576657274656400000000000000000000000000000000000000600082015250565b6000616223600d83614550565b915061622e826161ed565b602082019050919050565b600060608201905061624e6000830185613e42565b818103602083015261625f81616216565b905081810360408301526162738184614642565b90509392505050565b7f41413336206f766572207061796d6173746572566572696669636174696f6e4760008201527f61734c696d697400000000000000000000000000000000000000000000000000602082015250565b60006162d8602783614550565b91506162e38261627c565b604082019050919050565b60006040820190506163036000830184613e42565b8181036020830152616314816162cb565b905092915050565b600080fd5b600080fd5b6000808585111561633a5761633961631c565b5b8386111561634b5761634a616321565b5b6001850283019150848603905094509492505050565b600082905092915050565b60007fffffffffffffffffffffffffffffffffffffffff00000000000000000000000082169050919050565b600082821b905092915050565b60006163b18383616361565b826163bc813561636c565b925060148210156163fc576163f77fffffffffffffffffffffffffffffffffffffffff00000000000000000000000083601403600802616398565b831692505b505092915050565b60007fffffffffffffffffffffffffffffffff0000000000000000000000000000000082169050919050565b600061643c8383616361565b826164478135616404565b92506010821015616487576164827fffffffffffffffffffffffffffffffff0000000000000000000000000000000083601003600802616398565b831692505b505092915050565b7f414131302073656e64657220616c726561647920636f6e737472756374656400600082015250565b60006164c5601f83614550565b91506164d08261648f565b602082019050919050565b60006040820190506164f06000830184613e42565b8181036020830152616501816164b8565b905092915050565b7f4141313320696e6974436f6465206661696c6564206f72204f4f470000000000600082015250565b600061653f601b83614550565b915061654a82616509565b602082019050919050565b600060408201905061656a6000830184613e42565b818103602083015261657b81616532565b905092915050565b7f4141313420696e6974436f6465206d7573742072657475726e2073656e646572600082015250565b60006165b9602083614550565b91506165c482616583565b602082019050919050565b60006040820190506165e46000830184613e42565b81810360208301526165f5816165ac565b905092915050565b7f4141313520696e6974436f6465206d757374206372656174652073656e646572600082015250565b6000616633602083614550565b915061663e826165fd565b602082019050919050565b600060408201905061665e6000830184613e42565b818103602083015261666f81616626565b905092915050565b600060408201905061668c6000830185614b3a565b6166996020830184614b3a565b939250505056fea2646970667358221220f6e7d5acc3b3ce47e8c4e79583caee84134eab4d0c55f3973f51a15d53351cdb64736f6c63430008170033",
```

Comment:
For the sake of simplicity and maintenance, we could also use dummy code here with a minimal implementation of ERC-7821 so we're not testing real code in our tests?

Reply:
mmm I'm still tempted to use the bytecodes of the delegator contracts we use, as being one or another seems to be manageable in a similar way (we'll have bytecode which is still "un-readable", if that make sense).
In the platform team we'll work in the future on how to better manage network states and map that with contracts, to have a better control (and generation of those) but for now this is still an open topic, that needs to be think and defined. Would that sound good to you?